src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -84,12 +83,10 @@
      * This is sometimes referred to as the Range Raptor Search Window - but should apply to all
      * scheduled/time dependent travels.
      * <p>
-     * Optional - it is NOT recommended to set this value, unless you use the value returned by the
-     * previous search. Then it can be used to get the next/previous "page". The value is
-     * dynamically assigned a suitable value, if not set. In a small to medium size operation
-     * you may use a fixed value, like 60 minutes. If you have a mixture of high frequency cities
-     * routes and infrequent long distant journeys, the best option is normally to use the dynamic
-     * auto assignment.
+     * Optional - The value is dynamically assigned a suitable value, if not set. In a small to
```

Comment:
Again, should we comment on the relation with the paging automatically increasing the search window?

Reply:
I did it on the `searchWindowUsed`, but not here. There are several other parameters(dateTime, arriveBy) that are overridden as well - I think this become implementation details. The contract is, pass in the pre/next token and the same unchanged request to do the paging.